tools/pip/setup.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
```

Comment:
Are these terrifying looking python files just copied boilerplate from somewhere else?

Reply:
This is partially copied from the prior CLI python wrapper (used for 1.7 compat), which was also a `.whl`.